lib/tasks/email_checkout_reminder.rake

```diff
@@ -0,0 +1,19 @@
+desc 'Send email reminder about upcoming checkout'
```

Comment:
Let's leave it for now, we can have a wish list issue open to write tests for all of the mailman rake tasks and consolidate them into a single file.